### PR TITLE
Add notice on the frontend

### PR DESCRIPTION
**Pull request by sejas**
- Related: https://github.com/Automattic/dotcom-forge/issues/5513

## Description

Add notice in the front page of demo sites.
The notice can be closed and will write a cookie to avoid showing it next times for that user/site.
Clicking on Try  will open a new tab in the browser.

## Testing instructions

* Add this plugin file `studio-demo-site-companion.php` directly into the `wp-content/mu-plugins/` folder in any website. For example in Studio, wp-now or production. You'll have only one file in `wp-content/mu-plugins/studio-demo-site-companion.php`
* Navigate to `/`
* Observe the changes mentioned in the description
* Access `/wp-admin`
* Observe the notice and the new background color.

Screenshot 2024-04-22 at 15 54 16

Screenshot 2024-04-22 at 15 54 24

### CODE REVIEW

File: studio-demo-site-companion.php [hunk 1]
```diff
@@ -26,23 +24,27 @@ function studio_companion_admin_notices() {
...
-		}
-	</style>
-	<?php
-}
\ No newline at end of file
+function studio_companion_enqueue_scripts() {
+	wp_enqueue_style( 'studio-companion', plugin_dir_url( __FILE__ ) . 'assets/style.css' );
+	wp_enqueue_script( 'studio-companion', plugin_dir_url( __FILE__ ) . 'assets/index.js', array( ), '1.0', true );
+	wp_localize_script( 'studio-companion', 'studioCompanionNotice', array(
+		'description' => __( "You're previewing a <b>WP Build</b> site." ),
```
Comment (wojtekn) [inline studio-demo-site-companion.php]: WP Build? 😀 Can we match the design from the linked issue?

Comment (sejas) [reply studio-demo-site-companion.php]: Done!, I think I was using a different figma link.

**Review by katinthehatsite — APPROVED** [review]
It looks good on my test site. 

The only thing that I saw compared to the design is that the arrow is a bit bigger in the design and is aligned more towards the bottom (at the moment, the arrow is sort of starting in the "vertical middle"):

Screenshot 2024-04-22 at 10 06 28 AM

However, I am not sure how important this small change is so I am approving regardless 👍 Nice work!!

**Comment by wojtekn** [comment]
@sejas it looks good, but it won't work that way as we currently install the plugin by copying PHP file: D145907-code

Would it work to keep everything in the main PHP file?

**Comment by sejas** [comment]
@wojtekn , ha!, I didn't know that.
Let me check alternatives. I wonder if we should change D145907 and download the plugin zip and unzip it.

**Comment by sejas** [comment]
@wojtekn , @katinthehatsite , could you review it again?
I've updated the description to explain this plugin should be added to the mu-plugins folder.

File: studio-demo-site-companion.php (hunk 1 above)
Comment (wojtekn) [reply studio-demo-site-companion.php]: Thanks for updating those!

**Review by wojtekn — APPROVED** [review]

**Pull request merged**